tests/HttpClientConnectionManagerTest.cpp

```diff
@@ -113,7 +113,7 @@ static int s_TestHttpClientConnectionManagerResourceSafety(struct aws_allocator
     connections.clear();
     for (auto &connection : connectionsCpy)
```

Comment:
clearing connectionsCpy would have the same effect, wouldn't it?

Reply:
oh.... yeah true.